Resources/lu-Composition.xml

```diff
@@ -26,33 +26,36 @@
   <kind value="resource" />
   <abstract value="false" />
   <type value="Composition" />
-  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Composition" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/clinicaldocument" />
   <derivation value="constraint" />
   <differential>
     <element id="Composition">
       <path value="Composition" />
       <short value="Laboratory Report composition" />
       <definition value="A composition is a set of healthcare-related information that is assembled together into a single logical document that provides a single coherent statement of meaning, establishes its own context and that has clinical attestation with regard to who is making the statement. While a Composition defines the structure, it does not actually contain the content: rather the full content of a document is contained in a Bundle, of which the Composition is the first resource contained." />
     </element>
+    <element id="Composition.identifier">
+      <path value="Composition.identifier" />
+      <short value="Version-independent business identifier of the Laboratory Report (setID)" />
+      <definition value="A version-independent  Business identifier of the Laboratory Report (setID). This identifier stays constant as the Laboratory Raport may be updated and changed over time." />
+      <min value="1" />
+    </element>
     <element id="Composition.type">
       <path value="Composition.type" />
       <short value="Kind of composition (Laboratory Report)" />
       <definition value="Laboratory Report" />
-    </element>
-    <element id="Composition.type.coding">

```

Comment:
This element with id "Composition.type.coding" is removed and the <patternCodeableConcept> added to the <element> with id "Composition.type" - is this the intention or should the  <patternCodeableConcept> be added to the <element> with id "Composition.type.coding"?